python/instrumentation/openinference-instrumentation-litellm/src/openinference/instrumentation/litellm/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.7"
```

Comment:
maybe copy pasted? 0.0.1 for initial version makes more senes?

Reply:
Changed to 0.0.1